examples/framework.rs

```diff
@@ -1,6 +1,8 @@
 use std::future::Future;
 #[cfg(not(target_arch = "wasm32"))]
 use std::time::{Duration, Instant};
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen::prelude::*;
```

Comment:
nit: can we do it as the first thing in `fn run` (which is already under the target_arch cfg), to avoid extra cfg